### DISCUSSION TITLE
Cart Drawer Reload

### DISCUSSION THREAD

**Discussion by szykuc** [Shopify/dawn#3074] [Q&A]
Hello, 
how can I to force the cart drawer to be reload after an js event is called from a certain app?
I need exactly the same reload that takes place after changing the quantity of products - when the all cart drawer content is reloading.
Is it posible? I tried to force run updateQuantity function from cart.js but I know this is a poor approach ;)

**Comment by samuelroeland** [comment]
Hi dos-atelier.com can help you
